libs/payments/paypal/src/lib/client.ts

```diff
@@ -122,8 +121,49 @@ export class PayPalClient {
       return resultObj;
     } else {
       // TypeScript doesn't narrow ACK, necessitating a cast
-      throw new PayPalClientError(result.text, resultObj as NVPErrorResponse);
+      throw new MultiError(
+        this.getListOfPayPalClientErrors(
+          result.text,
+          resultObj as NVPErrorResponse
+        )
+      );
+    }
+  }
+
+  public getListOfPayPalClientErrors(raw: string, data: NVPErrorResponse) {
+    const errors: PayPalClientError[] = [];
+    if (!data.L || !data.L.length) {
+      const message =
+        'PayPal NVP returned a non-success ACK. See "this.raw" or "this.data" for more details.';
+      const err = new PayPalClientError(raw, data, {
+        message,
+      });
+      errors.push(err);
+      return errors;
+    }
+
+    // Sort errors in alphabetical order by SEVERITYCODE
+    // This puts type "Error" before type "Warning".
+    const nvpErrors = data.L.sort((errA, errB) => {
```

Comment:
In the original `PayPalClientError` implementation, this was a `filter` that would remove any warning messages instead of a `sort`. One benefit of the `MultiError` class is we don't have to pick which message to keep. We can keep all of them.